client/dashboard/sites/logs/dataviews/index.tsx

```diff
@@ -259,6 +320,15 @@ function SiteLogsDataViews( {
 					header={ LogsHeader }
 				/>
 			) }
+			{ showScrollTop && (
+				<Button
+					variant="tertiary"
+					className="site-logs-scroll-to-top"
```

Comment:
It would be better to assign the icon with default variant for the icon button

```js
__next40pxDefaultSize
size="compact"
icon={ arrowUp }
iconSize={ 24 }
```